### PR TITLE
fix: make full_chain_odd.py really work with pythia8+Geant4, add test

### PR DESCRIPTION
**Pull request by benjaminhuth**
- Fixes some minor issues in the `full_chain_odd.py` when run it with pyhia8+geant4.
   - currently there are FPEs, but I think it is okay to disable this for now and fix that later
- Adds `argeparse` command line arguments to steer number of events, simulation and ttbar
- Adds test function for 1 event to the `pytest` (caveat: 1 event takes **10 minutes** on my local machine...)

cc @andiwand @paulgessinger @pbutti

### CODE REVIEW

File: Examples/Python/tests/test_examples.py [hunk 1]
```diff
@@ -1225,7 +1253,7 @@ def test_full_chain_odd_example(tmp_path):
     env["NEVENTS"] = "1"
     env["ACTS_LOG_FAILURE_THRESHOLD"] = "WARNING"
```
Comment (andiwand) [inline Examples/Python/tests/test_examples.py:1254]: should we kill those?

**Review by andiwand — COMMENTED** [review]
looks good. lets see how long the ci takes

**Comment by pbutti** [comment]
Hi @benjaminhuth,

thank you for these updates. Is there a way to also add the download of the G4 data files to the geant4.sh setup script (or to another setup script) ?

File: Examples/Python/tests/test_examples.py (hunk 1 above)
Comment (benjaminhuth) [reply Examples/Python/tests/test_examples.py]: yes

**Comment by benjaminhuth** [comment]
I think we could try to add a check if the downloaded data of Geant4 are present. 
But not sure if we should try to download them inside this script.

**Comment by pbutti** [comment]
In principle the check is there (as it will return errors for data missing). What would be good to have is the script that at least tells you to issue ```download_geant4_data.sh``` if the tables are missing

**Comment by paulgessinger** [comment]
> In principle the check is there (as it will return errors for data missing). What would be good to have is the script that at least tells you to issue `download_geant4_data.sh` (when running in the docker) if the tables are missing

Btw I recently learned G4 actually ships with this functionality: `/usr/local/bin/geant4-config --install-datasets`.
So we don't need to use this extra script I need anymore.

File: Examples/Python/tests/test_examples.py [hunk 2]
```diff
@@ -1224,10 +1224,37 @@ def test_full_chain_odd_example(tmp_path):
...
+    script = (
+        Path(__file__).parent.parent.parent.parent
+        / "Examples"
+        / "Scripts"
+        / "Python"
+        / "full_chain_odd.py"
+    )
+    assert script.exists()
+    env = os.environ.copy()
+    env["ACTS_LOG_FAILURE_THRESHOLD"] = "WARNING"
```
Comment (benjaminhuth) [inline Examples/Python/tests/test_examples.py:1255]:
```suggestion
    env["ACTS_LOG_FAILURE_THRESHOLD"] = "ERROR"
```

**Review by andiwand — APPROVED** [review]
:rocket:

**Pull request merged**